src/api/spec/support/database_cleaner.rb

```diff
@@ -11,18 +12,23 @@
     # and is seeded
     load "#{Rails.root}/db/seeds.rb"
     Rails.logger.level = log_level
+    # Truncate the users and roles_user loaded in db/seeds.rb to be consistent
```

Comment:
But all the others tables are empty because we have just clean de database and loaded the seeds.